### PR TITLE
cleanup docs a bit + fix some broken links

### PR DESCRIPTION
**Pull request by 0xdevalias**
Cleaned up some of the docs a bit. Follow on from:

- https://github.com/sparkletown/sparkle/pull/1611

partially closes sparkletown/internal-sparkle-issues#778

### CODE REVIEW

File: docs/bootstrap-application.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-### Connect your application to Firebase
+# Connect your application to Firebase
+
 ## 'Bootstrap' the platform
 
 Make sure that you're located in ./scripts folder.
```
[the remark this answers is not in the document]
Comment (sunny-viktoryia) [reply docs/bootstrap-application.md:5]: Line10 (and below where needed): add `bash` after the triple backticks.

File: docs/getting-started.md [hunk 2]
```diff
@@ -4,31 +4,33 @@
...
 
-You can register a new user by pressing Log In in the top right corner:
+You can register a new user by pressing 'Log In' in the top right corner:
 
-Once redirected to authorization form select Create account:
-* Fill in necessary data and press Continue
+Once redirected to authorization form select 'Create account':
 
-* Fill in your Username and upload custom avatar, or select from the default ones. Then press Create my profile:
+* Fill in necessary data and press Continue
```
Comment (sunny-viktoryia) [inline docs/getting-started.md:28]:
```suggestion
* Fill in necessary data and press _Continue_
```

File: docs/twilio-configuration.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ Login to Twilio
```
Comment (sunny-viktoryia) [inline docs/twilio-configuration.md:34]: a bit more polishing: replace `'text'` with `_text_`

File: docs/create-new-environment.md [hunk 4]
```diff
@@ -18,6 +19,7 @@ This part of the setup is complete!
 
 
 ### Step 2: Configure Firebase Project Settings
+
 1. Go to https://console.firebase.google.com/ and find the `Example Project` you chose in step 1
 
 2. From _Project Overview_, hover over the gear icon and click _Project Settings_
```
[the remark this answers is not in the document]
Comment (0xdevalias) [reply docs/create-new-environment.md:25]: Calling this OOS. Honestly I want to do a pass through all of the docs and change/clean up the style of them when I get a chance. Not super happy with it overall, but it's functional enough for now.

File: docs/create-new-environment.md [hunk 5]
```diff
@@ -48,6 +50,7 @@ This part of the setup is complete!
 This part of the setup is now complete!
 
 ### Step 4: Set up Firebase Hosting
+
 1. From the Firebase console, within the appropriate project, click on _Hosting_ on the left hand menu.
 
 2. Click on _Get started_
```
[the remark this answers is not in the document]
Comment (0xdevalias) [reply docs/create-new-environment.md:56]: Calling this OOS. Honestly I want to do a pass through all of the docs and change/clean up the style of them when I get a chance. Not super happy with it overall, but it's functional enough for now.

File: docs/create-new-environment.md [hunk 6]
```diff
@@ -132,20 +135,19 @@ npx firebase use TODO-PROJECT-ID
...
+Now we need to upload function config to Firebase. Use the `.json` file with private key that you've downloaded just recently:
 
-```
+```bash
 ./upload-function-config-service-account.ts TODO-PROJECT-ID example-project-firebase-adminsdk-XXXXX-XXXXXXXXXX.json
 ```
 
-```
-
+```bash
```
[the remark this answers is not in the document]
Comment (0xdevalias) [reply docs/create-new-environment.md:150]: Calling this OOS. Honestly I want to do a pass through all of the docs and change/clean up the style of them when I get a chance. Not super happy with it overall, but it's functional enough for now.

File: docs/create-new-environment.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (sunny-viktoryia) [reply docs/create-new-environment.md:25]: fair. Yep, that's why I approved the PR and just leaved a few comment for future reference. I'm good if we merge what we have right now.

**Comment by 0xdevalias** [comment]
@sunny-viktoryia I've accepted the suggested changes, but i'm not too worried about perfectly polishing the whole docs in this PR. I'd rather get the bulk of the changes out sooner, then can follow up with another one in future if needs be.

> Honestly I want to do a pass through all of the docs and change/clean up the style of them when I get a chance. Not super happy with it overall, but it's functional enough for now.
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1818#discussion_r668792437_

**Comment by 0xdevalias** [comment]
@alexey-petrov If there is anything in particular you wanted included here before I click merge, let me know, otherwise LGTM!

**Pull request merged**